_security/multi-tenancy/multi-tenancy-config.md

```diff
@@ -149,3 +149,93 @@ The Security plugin scrubs these index names of special characters, so they migh
 {: .tip }
 
 To back up your OpenSearch Dashboards data, [take a snapshot]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/) of all tenant indexes using an index pattern such as `.kibana*`.
+
+## `kibana_server` role details
+
+`kibana_server` role is used by OpenSearch Dashboards to perform necessary operations on OpenSearch. By default `kibanauser` is mapped to this role via `role_mappings.yml`. You can view the full list of permissions assigned to this role by running `GET` request on `_plugins/_security/api/roles/kibana_server` API using admin certificate, key and certificate authority file.
```

Comment:
```suggestion
OpenSearch Dashboards uses the`kibana_server` role to perform necessary operations on OpenSearch. By default `kibanauser` is mapped to this role through the `role_mappings.yml` file. You can view the full list of permissions assigned to this role by running the GET request on `_plugins/_security/api/roles/kibana_server` API using the admin certificate, key, and certificate authority file.
```

Reply:
Edited suggestion. Is "on the...API" the correct wording? The last sentence is a bit of a run-on.